storage-account-staging.tf

```diff
@@ -9,7 +9,7 @@ locals {
 
   // for each client service two containers are created: one named after the service
   // and another one, named {service_name}-rejected, for storing envelopes rejected by bulk-scan
-  client_service_names_stg = ["bulkscanauto", "bulkscan", "sscs", "divorce", "probate", "finrem", "cmc"]
+  client_service_names_stg = ["bulkscanauto", "bulkscan", "sscs", "divorce", "probate", "finrem", "cmc", "publiclaw", "privatelaw"]
```

Comment:
after meeting with nfd, we can add nfdivorce.

Reply:
and we need same for reform-scan

Reply:
sorry, already raised: https://github.com/hmcts/reform-scan-shared-infra/pull/101
missed to assign it to you